### PR TITLE
[8.0] [ADD] web_image_viewer: View images directly in web client

### PR DESCRIPTION
**Pull request by hoangpq**
Help me to merge pull request for create new module which allow user to view image directly in odoo inbox or somewhere without download to local store

Module name: web_image_viewer

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Hi, Hoan, thanks for your contribution. The feature seems interesting indeed, but maybe people don't review this because of the version is a bit outdated or they are not able. You can encourage to review your PR if you review other PRs and ask for it in exchange.

I have changed the base branch, because you have proposed the module to 10.0 branch, which was incorrect, and also change title to a more proper one.

I'm making a quick code review, but not definitive.

File: web_image_viewer/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+View image in odoo without download to local store
+=============================
```
Comment (pedrobaeza) [inline web_image_viewer/README.rst:7]: The line needs to have the same length as the title.

File: web_image_viewer/README.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+This module was written to view image directly in odoo web client 
+without download to local store
```
Comment (pedrobaeza) [inline web_image_viewer/README.rst:10]: s/download/downloading

File: web_image_viewer/README.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Usage
+=====
+    Install addon [Settings > Modules > Install Modules]
```
Comment (pedrobaeza) [inline web_image_viewer/README.rst:14]: These are not usage instructions. Specify where you can find an example for seeing an image.

File: web_image_viewer/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    'name': 'Image Viewer',
+    'author': 'Hoang Phan, pquochoang2007@gmail.com',
+    'category': 'Hidden',
+    'version': '1.0',
```
Comment (pedrobaeza) [inline web_image_viewer/__openerp__.py:5]: Use 8.0.1.0.0

File: web_image_viewer/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    'description': """ """,
```
Comment (pedrobaeza) [inline web_image_viewer/__openerp__.py:6]: Don't put description key, as you already have README.rst.

File: web_image_viewer/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    'depends': ['base', 'web'],
+    'auto_install': True,
```
Comment (pedrobaeza) [inline web_image_viewer/__openerp__.py:8]: Don't make it auto-install

File: web_image_viewer/views/image_viewer.xml [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+<openerp>
+    <data>
+        <template id="image_viewer" name="document assets" inherit_id="web.assets_backend">
+            <xpath expr="." position="inside">
+                <script type="text/javascript" src="/image_viewer/static/lib/fancybox/source/jquery.fancybox.js" charset="utf-8"></script>
+                <script type="text/javascript" src="/image_viewer/static/src/js/viewer.js" charset="utf-8"></script>
+            </xpath>
+        </template>
+    </data>
+</openerp>
```
Comment (pedrobaeza) [inline web_image_viewer/views/image_viewer.xml:12]: Line at the end

File: web_image_viewer/static/src/js/viewer.js [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+                    });
+                });
+            });
+        },
+        display_attachments: function () {
+            this._super();
+            _.defer(_.bind(this.load_base64, this));
+        }
+    });
+}
```
Comment (pedrobaeza) [inline web_image_viewer/static/src/js/viewer.js:78]: Line at the end

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: web_image_viewer/static/src/js/viewer.js (hunk 8 above)
Comment (hoangpq) [reply web_image_viewer/static/src/js/viewer.js:78]: Thanks for your feedback, i will migrate this module to v10

Regards.

**Pull request closed without merging**